src/ui/mapper.ui

```diff
@@ -696,7 +696,23 @@
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
-               <string>Room IDs</string>
+               <string>IDs</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QCheckBox" name="showRoomNames">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="layoutDirection">
```

Comment:
Not sure this property is needed - and it might be problematic if anyone tries to add support for Arabic (or is it Hebrew) locales which use `RightToLeft` - I'd manually cut it out of the file (you might be able to undo it in the Qt Designer plugin/utility by clicking the small left pointing arrow on the extreme right of the option which should reset any option back to the default - and cause it to loose the boldness in its listing in that tool:

Look at the layoutDefault setting for this control (which won't be in `.ui` file):
![image](https://user-images.githubusercontent.com/6163092/88406868-2caedf80-cdc9-11ea-9912-d053c22bbddc.png)

See how it has been bolded after I change it but put it back to the default (for my current locale as it happens):
![image](https://user-images.githubusercontent.com/6163092/88407151-97f8b180-cdc9-11ea-85d2-af987b44521c.png)

However clicking on that (now red) arrow reverts/resets it (and disables the arrow and turns that grey):
![image](https://user-images.githubusercontent.com/6163092/88407369-eefe8680-cdc9-11ea-9a49-a753ec4cf2fa.png)

Reply:
Yeah, mis-click in the designer. I'll go through the designer files' changes and fix these parts.